cluster-autoscaler/simulator/dynamicresources/utils/utilization_test.go

```diff
@@ -141,7 +142,41 @@ func TestDynamicResourceUtilization(t *testing.T) {
 			wantHighestUtilization:     0.2,
 			wantHighestUtilizationName: apiv1.ResourceName(fmt.Sprintf("%s/%s", fooDriver, "pool1")),
 		},
+		{
+			testName: "AdminAccess ResourceClaim don't count for utilization",
+			nodeInfo: framework.NewNodeInfo(node,
+				testResourceSlices(fooDriver, "pool1", "node", 0, 10, 1),
+				testPodsWithClaims(fooDriver, "pool1", "node", 6, 1, ptr.To(true))...,
+			),
+			wantUtilization: map[string]map[string]float64{
+				fooDriver: {
+					"pool1": 0,
+				},
+			},
+			wantHighestUtilization:     0,
+			wantHighestUtilizationName: apiv1.ResourceName(fmt.Sprintf("%s/%s", fooDriver, "pool1")),
+		},
+		{
+			testName: "AdminAccess ResourceClaim don't count for utilization",
```

Comment:
The test name is the same as for the previous case - can you distinguish it? The difference is this one is testing multiple Pods with different admin access values, and the other just a single Pod with admin access enabled right?

Reply:
Thanks!